packages/fx-core/templates/plugins/resource/aad/auth/V3/README.txt

```diff
@@ -1,19 +1,23 @@
-// Draft. Needs review from yu.
-Enable single sign-on for Teams applications
+//Full content, plan to be put in Github wiki page
```

Comment:
If you want to put this file in wiki, you need to delete this file.
Github wiki supports markdown, we can use markdown instead of txt

Reply:
I just provide the content in this README file, after the content move to wiki, this file can be delete.